### PR TITLE
Android x migration

### PR DESCRIPTION
**Pull request by f7deleon**
- Android x migration of the project

### CODE REVIEW

File: fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java [hunk 1]
```diff
@@ -15,17 +15,20 @@
...
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.Executor;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import androidx.annotation.AnyThread;
+import androidx.annotation.GuardedBy;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import androidx.paging.DataSource;
```
Comment (FerrariBruno) [inline fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java]: can you check? this wasn't a previous import

File: fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java [hunk 2]
```diff
@@ -15,17 +15,20 @@
...
+import androidx.paging.PagedList;
```
Comment (FerrariBruno) [inline fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java]: same

File: fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java (hunk 1 above)
Comment (matir91) [reply fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java]: we should update the detekt version, the new version checks it!

File: gradle.properties [hunk 3]
```diff
@@ -11,3 +11,5 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+android.useAndroidX=true
+android.enableJetifier=true
```
Comment (matir91) [inline gradle.properties]: needed ? why?

**Review by mirland — APPROVED** [review]

**Pull request closed without merging**